package/base-files/files/sbin/firstboot

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-/sbin/jffs2reset $@
+/sbin/jffs2reset "$@"
```

Comment:
First commit looks simple and reasonable.  The subject line (first line used by shortlog) should be prefixed with "base-files:".